container-engine-lib/lib/backend_interface/objects/service_directory/persistent_directories.go

```diff
@@ -17,3 +37,7 @@ func NewPersistentDirectories(persistentDirectories map[string]PersistentDirecto
 		ServiceDirpathToPersistentDirectory: persistentDirectories,
 	}
 }
+
+func IsPersistentKeyValid(persistentKey DirectoryPersistentKey) bool {
```

Comment:
I like that you put the validation inside the object, good way to centralize this logic.